templates/stripes/stripe-table.php

```diff
@@ -0,0 +1,55 @@
+<?php
+$caption = !empty($caption["html"]) ? $this->inline($caption["html"]) : null;
+$headersInFirstRow = !empty($headersInFirstRow) ? $headersInFirstRow : false;
+$headersInFirstColumn = !empty($headersInFirstColumn) ? $headersInFirstColumn : false;
+
+// Ensure each item in the dataset is an array.
+$dataset = !empty($dataset) && is_array($dataset) ? $dataset : [[]];
+foreach ($dataset as $key => $value) {
+    if (!is_array($value)) {
+        $dataset[$key] = [];
+    }
+}
```

Comment:
Add a big check after all of this (and probably an `array_filter`) for whether the `$dataset` is empty? It looks like we shouldn't render anything if it is.

Reply:
And—you could potentially (should??) do some of this filtering before you even get to the template.

Reply:
meaning in the frontend admin? I can add some checks there too 🙆‍♂️